### ISSUE TITLE
Symfony 3.2.4 FTP Adapter is broken

### ISSUE THREAD

**Issue by sebastianblum** [1up-lab/OneupFlysystemBundle#129]
hello!

I'm using version 1.10.0 and today with the symfony upgrade to 3.2.4 I got this exception on cache:clear command:

```
sebastianblum$ bin/console cache:clear -vvv

                                                                                  
  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]   
  Unrecognized option "ftp" under "oneup_flysystem.adapters.my_nas_ftp"  
                                                                                  

Exception trace:
 () at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/ArrayNode.php:317
 Symfony\Component\Config\Definition\ArrayNode->normalizeValue() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/BaseNode.php:264
 Symfony\Component\Config\Definition\BaseNode->normalize() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/PrototypedArrayNode.php:283
 Symfony\Component\Config\Definition\PrototypedArrayNode->normalizeValue() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/BaseNode.php:264
 Symfony\Component\Config\Definition\BaseNode->normalize() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/ArrayNode.php:307
 Symfony\Component\Config\Definition\ArrayNode->normalizeValue() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/BaseNode.php:264
 Symfony\Component\Config\Definition\BaseNode->normalize() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Processor.php:33
 Symfony\Component\Config\Definition\Processor->process() at .../vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Processor.php:50
 Symfony\Component\Config\Definition\Processor->processConfiguration() at .../vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Extension/Extension.php:99
 Symfony\Component\DependencyInjection\Extension\Extension->processConfiguration() at .../vendor/oneup/flysystem-bundle/DependencyInjection/OneupFlysystemExtension.php:26
 Oneup\FlysystemBundle\DependencyInjection\OneupFlysystemExtension->load() at .../vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php:59
 Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationPass->process() at .../vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/MergeExtensionConfigurationPass.php:39
 Symfony\Component\HttpKernel\DependencyInjection\MergeExtensionConfigurationPass->process() at .../vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php:120
 Symfony\Component\DependencyInjection\Compiler\Compiler->compile() at .../vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:565
 Symfony\Component\DependencyInjection\ContainerBuilder->compile() at .../vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:484
 Symfony\Component\HttpKernel\Kernel->initializeContainer() at .../vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:116
 Symfony\Component\HttpKernel\Kernel->boot() at .../vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:68
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at .../vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:120
 Symfony\Component\Console\Application->run() at .../bin/console:29
```

The configuration is the default one from your bundle documentation.

Downgrading to Symfony 3.2.3 solves the problem again.

greetings, Sebastian

**Comment by bytehead** [comment]
Thanks for reporting. 
Symfony `3.2.4` or `2.3.4`?

**Comment by sebastianblum** [comment]
sorry @bytehead the 3.2.4 version which was released today. see http://symfony.com/blog/symfony-3-2-4-released

**Comment by thanosp** [comment]
Same here. Breaks after update to symfony v3.2.4.
It's not just the ftp adapter though but probably all.

**Comment by Sander-Toonen** [comment]
I have the same problem. 1.9.0 works fine, but after upgrading to 1.10.0 all adapters give the same error. Even the 'local' adapter:

      [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]
      Unrecognized option "local" under "oneup_flysystem.adapters.local_adapter"

**Comment by bytehead** [comment]
See #130.

**Comment by bytehead** [comment]
Fixed in 6885894. Thanks to @chosroes! :tada:

**Comment by bytehead** [comment]
There is also a new version available: [1.10.1](https://github.com/1up-lab/OneupFlysystemBundle/releases/tag/1.10.1)

**Comment by chrishodgsonplt** [comment]
I have the same problem giving error `Unrecognized option "client" under "oneup_flysystem.adapters.catalog_storage_adapter.awss3v3"`

in `composer.lock`:
symfony/symfony: v3.4.2
oneup/flysystem-bundle: 1.14.0
aws/aws-sdk-php: 3.52.31
league/flysystem-aws-s3-v3: 1.0.40

in `services.yml`:
services:
  app.amazon.s3_client:
    class: Aws\S3\S3Client
    arguments:
      -
        credentials:
          key: MY_KEY
          secret: MY_SECRET
        version: latest
        region: eu-west-1

in `config.yml`:
oneup_flysystem:
  adapters:
    catalog_storage_adapter:
      awss3v3:
        client: app.amazon.s3_client
        bucket: ~
        prefix: ~
